dts/x86/mem.h

```diff
@@ -0,0 +1,9 @@
+#ifndef __DT_BINDING_ST_MEM_H
+#define __DT_BINDING_ST_MEM_H
+
+#define __SIZE_K(x) (x * 1024)
+
+#define DT_FLASH_SIZE		__SIZE_K(144)
+#define DT_SRAM_SIZE		__SIZE_K(55)
+
```

Comment:
Isn't these valid for arduino only? (I am asking that because the file seems generic to all x86)